contracts/reserve-auction/src/instantiate.rs

```diff
@@ -21,15 +22,76 @@ pub fn instantiate(
 ) -> Result<Response, ContractError> {
     set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
     let config = Config {
-        marketplace: deps.api.addr_validate(&msg.marketplace)?,
-        min_reserve_price: msg.min_reserve_price,
-        min_bid_increment_pct: Decimal::percent(msg.min_bid_increment_bps),
+        fair_burn: deps.api.addr_validate(&msg.fair_burn)?,
+        trading_fee_pct: bps_to_decimal(msg.trading_fee_bps),
```

Comment:
```suggestion
        trading_fee_pct: Decimal::bps(msg.trading_fee_bps),
```

Reply:
Double check this but might be able to use the new `Decimal::bps` that I added to CW.